stdlib/Profile/src/Profile.jl

```diff
@@ -990,7 +990,7 @@ function tree_format(frames::Vector{<:StackFrameTree}, level::Int, cols::Int, ma
                     link = editor_link(path, li.line)
                     styled"{link=$link:$styled_path}"
                 end
-                strs[i] = Base.annotatedstring(stroverhead, "╎", base, strcount, " ", rich_file, "; ", fname)
+                strs[i] = Base.annotatedstring(stroverhead, "╎", base, strcount, " ", rich_file, "  ", fname)
```

Comment:
This commit broke BenchmarkTools. What is the intended fix, here or in BenchmarkTools?
https://github.com/JuliaCI/BenchmarkTools.jl/actions/runs/11898803449/job/33156070523

Reply:
Replace the `;` with a space in the regex in BenchmarkTools?